requirements/docs.txt

```diff
@@ -1,5 +1,6 @@
 Pillow~=9.5
-Sphinx~=7.2
+Sphinx~=7.4
 sphinx_gallery~=0.14
-sphinx_rtd_theme~=1.3
+sphinx_rtd_theme~=2.0
+sphinx_substitution_extensions==2024.8.6
```

Comment:
Oh wow, what is this?

Reply:
Ah, that makes it so that Sphinx can do variable substitution in code blocks (so it lets us get rid of a bunch of stuff in conf.py)